scrapers/Spacebattles/Spacebattles/spiders/spacebattles_scraper.py

```diff
@@ -0,0 +1,191 @@
+import scrapy
+from datetime import datetime
+import sys
+import os
+import re
+from django.conf import settings
+from django.db import connection
+import django
+
+
+sys.path.insert(0, "../../website")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+# setup_environ(settings)
+django.setup()
+
+from noveltorpedo.models import *
+
+
+class StorySpider(scrapy.Spider):
+    name = "stories"
+    priority = 1
+    host = "spacebattles.com"
+
+    thread_queue = []
+
+    def start_requests(self):
+        urls = [
+            "https://forums.spacebattles.com/forums/creative-writing.18"
+        ]
+
+        for url in urls:
+            yield scrapy.Request(url=url, callback=self.loop_pages)
+
+    def loop_pages(self, response):
+            
+        """ loop through the pages on the main page
+
+        Keyword arguments:
+        response -- the response object used to navigate the page
+
+        """
+
+        current_page = response.xpath("//a[@class='currentPage ']/text()")
+        print("current page: {0}".format(current_page.extract_first()))
+
+        next_page_link = response.xpath("//a[@class='text' and contains(., 'Next')]")
+        next_page_link = next_page_link.xpath('@href').extract_first()
+
+        # urls_stories is a touple with a url, and a corresponding Story object
+        urls_stories = self.get_thread_urls(response)
+
+        for (url, story) in urls_stories:
+            yield scrapy.Request(url, callback=self.scan_thread, priority=0, meta={"story_item": story})
+
+        # yield scrapy.Request(urls[0], callback=self.scan_thread)
+        
+        # TODO: ADD CHECK HERE to only proceed when the thread_queue is empty!!!
+        """
+        if next_page_link is not None:
+
+            #print("next page link: {0}".format(next_page_link))
+            next_page_link = response.urljoin(next_page_link)
+            yield scrapy.Request(next_page_link, callback=self.loop_pages, priority=0)
+        """
+
+    def get_thread_urls(self, response):
+
+        """ Scan the threads on a page
```

Comment:
Reminding you that you wanted to update this docstring (and maybe do a pass at other docstrings).